brave/mumble/service.py

```diff
@@ -162,7 +163,7 @@ def authenticate(self, name, pw, certificates, certhash, certstrong, current=Non
                 # Define the registration date if one has not been set.
                 Ticket.objects(character__name=name, registered=None).update(set__registered=datetime.utcnow())
         
-            for tag in ('member', 'blue', 'guest', 'mumble'):
+            for tag in ('admin', 'member', 'blue', 'guest', 'mumble'):
```

Comment:
Do we need more than https://github.com/kiu/mumble/blob/kiu-cleanup-production/permissions.txt ?